CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.1.0
+* Support Rails 7.1
+* Drop support for Rails < 7
```

Comment:
with the change to `~> 7.1` in the `.gemspec` it seems like we are dropping support for 7.0 also?

Reply:
updated